Sources/Clappr_tvOS/Classes/Base/DecoratedPressAVPlayerViewController.swift

```diff
@@ -0,0 +1,31 @@
+import AVKit
+
+class DecoratedPressAVPlayerViewController: AVPlayerViewController {
+    let clapprPlayer: Player
+
+    init(player: Player) {
+        self.clapprPlayer = player
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    public required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private var isPaused: Bool {
+        return clapprPlayer.activePlayback?.isPaused ?? false
+    }
+
+    override func pressesBegan(_ presses: Set<UIPress>, with event: UIPressesEvent?) {
+        if presses.containsAny(pressTypes: [.select, .playPause]) && isPaused {
+            clapprPlayer.activePlayback?.trigger(.willPlay)
```

Comment:
I think we are not triggering `willPause` event when playback is paused by remote